src/Raven.Client/Documents/Operations/ETL/SQL/SqlConnectionStringParser.cs

```diff
@@ -24,11 +23,70 @@ internal static (string Database, string Server) GetDatabaseAndServerFromConnect
                     database = GetConnectionStringValue(connectionString, new[] { "Database" });
                     server = GetConnectionStringValue(connectionString, new[] { "Host", "Data Source", "Server" });
 
-                    var port = GetConnectionStringValue(connectionString, new[] { "Port" });
+                    var postgrePort = GetConnectionStringValue(connectionString, new[] { "Port" });
 
-                    if (string.IsNullOrEmpty(port))
-                        server += $":{port}";
+                    if (string.IsNullOrEmpty(postgrePort))
+                        server += $":{postgrePort}";
                     break;
+                case SqlProvider.MySqlClient:
```

Comment:
Do we really need to parse the connection strings? Why not pass them as is to the `AdoConnection`?

Reply:
This class (and its friends that use it) are in Raven.Client where we don't have ADO dependencies. The values that we parse here are displayed in the studio (ongoing task view) and go into the logs (`dbname@server`).  When we do the actual connection then we pass the entire connection string.